action.yml

```diff
@@ -7,12 +7,15 @@ branding:
 
 inputs:
   python-version: 
-    description: 'Override version of python used to run ape- default 3.8'
+    description: 'Override version of python used to run ape- default 3.10'
     required: False
-    default: '3.8'
+    default: '3.10'
   ape-version-pin:
     description: 'Override version of eth-ape'
     required: False
+  ape-remote-git-version:
```

Comment:
im wondering if instead we want to allow `git+` values in `ape-version-pin`, so it works the same as https://github.com/ApeWorX/ape/pull/1468

Reply:
I don't see why not, you could probably just a switch on where it starts with `==`/`>=`/etc.